### ISSUE TITLE
Interview Priority Analysis: 37

### ISSUE THREAD

**Issue by raquelnish** [codeforamerica/brigade-playbook#108]
### Overview
For UXR team, we need to process & analyze Interview number 37 in order to complete our analysis phase. 

### Instructions for working this issue
Do the action items defined by the checkboxes. Each numbered step is a discrete task. We encourage you to complete all the tasks for this brigade's interview, however, you can stop after any of the numbered tasks.

### Action Items

- [ ] This brigade interview needs the following work done:
     - [ ] 1. Transcription verification
          - [ ] Add data organization label & remove transcription verification label.
     - [ ] 2. Data organization
          - [ ] Add coding label & remove data organization label.
     - [ ] 3. Coding
          - [ ] Add tagging label & remove coding label.
     - [ ] 4. Tagging

### Reporting progress
- Be sure to check the box when you finish an action item.
- Report your progress weekly in a comment below using the following format:
     - Progress: 
     - Blockers: 
     - Availability :
     - Estimated date of completion:

#### Reasons: 
- Got personally busy, can’t finish
-  Only want to do a specific type of task per interview
- There is a blocker (e.g. instructions not ready yet) in the way of finishing and I still have time to work on other issues 
#### What to do if you have to stop working mid issue:
- Note your progress in the checkboxes 
- Add a note in the comments with details
- Move the issue to the questions/review column 

### Resources/Instructions
[Interview Data Sheet](https://docs.google.com/spreadsheets/d/1p1Arx4ZFB9ixCP81FmkNRAK8iiiEtsgab67gjiVbp6o/edit#gid=1783476144)
[Transcription Verification Instructions](https://docs.google.com/document/d/1kTrslqYY0whUjCMxbcV7Ft1nzCr_UTPWXfhomNgOvEw/edit)
[Data Organization Instructions](https://docs.google.com/document/d/1vJqDQ8Th8kSgisSSiMzDpRHTIWI3ILCX4eOTgMwRdoY/edit)
[Coding Instructions](https://docs.google.com/document/u/0/d/10am6Ki0gJKzITt2kcjMsR233oC_9WJneKnFimFc8Mao/edit)
[Tagging Instructions](https://docs.google.com/document/d/1soNmLMyVspkgz2MMD5DuF_WVBBmnFLFgCPNZDjxjDWI/edit#)

**Comment by Baleja** [comment]
Progress:
Blockers: Hey there I am stuck on: Instructions for Data organization question 11 & 12
Availability: Approx 6 hrs
ETA: soon!

**Comment by Baleja** [comment]
Make sure column F is filled with pasted quotes to continue to the next stage.

**Comment by Baleja** [comment]
Still having a little difficulty getting from data organisation to coding. The prompt boxes for Google colab do not come up when doing a second time :'(

**Comment by MarianneAnthonette** [comment]
@MarianneAnthonette aka me: I'll audit the transcription sheet (and make notes for Serena for next time) and fix the data organization part as well on Friday.

Don't start coding until I send a message back to you.

**Comment by MarianneAnthonette** [comment]
@Baleja Yeah, I don't have time to finish fixing the transcription sheet. Go back to the sheet and make the following edits:

- I went through the Actual cleaned input column and moved text around (you missed the Code of Conduct question). You need to **transcribe the entire interview, so you also have to transcribe row 114 and onward. The response in row 123 seems crucial and needs to be moved into the additional topics response row**.
- I went through and **deleted the timestamps and some capitalization fixes, but you have to do this for all the remaining errors**.
- I did fix the transcription and separating out ideas for a few questions (you can tell if the data is in col F or onward). **This means that they don't need to be worked on**. Use this a reference.
- When you separate out ideas, **make sure you don't remove text that provides context.** The goal is just to break the text apart. Leave it in there. A lot of your previous data organization deleted a lot of crucial text (it's duplicated in the Old Transcription tab).

Focus on fixing the transcription sheet and separating the ideas, and message me back.

**Comment by bdavis73** [comment]
Transcript verification is complete. Updated status and label.
